README.md

```diff
@@ -1,28 +1,182 @@
 # Velveteen [![Build Status](https://travis-ci.com/thoughtbot/velveteen.svg?branch=master)](https://travis-ci.com/thoughtbot/velveteen)
 
-🚧 A work in-progress 🚧
-
 Transform your background jobs into a real data pipeline with Velveteen.
 
-## Install
+## We've got a pipeline problem
```

Comment:
This feels like a much bigger thing than we're giving here. As a Rails dev who's built background jobs, I don't truly have an understanding of *why* I want a pipeline or what it does. What does it give me that my stick tower of background jobs calling other jobs doesn't?

Maybe I just want to acknowledge that this header is the title of the blog post announcing Velveteen and that I want this section of the README to sum up its points or link to it.

Reply:
I do think this header would be better used as a blog post title, which would in turn be a good place to lay out some examples of the background job pain that Velveteen is trying to eliminate. For now, I think I'll only change the header here. Perhaps the wording can be improved by pulling things from the future blog(s).